models/tokens/optimism/tokens_optimism_erc20_bridged_mapping.sql

```diff
@@ -23,38 +23,39 @@ SELECT l1_token, l2_token
         ORDER BY COALESCE(et.decimals, map.decimals) ASC, COALESCE(map.symbol, et.symbol) DESC NULLS LAST) AS rnk
 FROM (
 
-        SELECT _l1Token AS l1_token, _l2Token AS l2_token, NULL AS symbol, NULL AS decimals
-            FROM {{source( 'optimism_ethereum', 'L1StandardBridge_evt_ERC20DepositInitiated' ) }}
-        WHERE evt_tx_hash != '0x460965f169a99b3d372cd749621a3652ad232d1b1580fd77424eacc2e973b672' --bad event emitted
-        {% if is_incremental() %}
-        AND evt_block_time >= date_trunc("day", now() - interval '1 week')
-        {% endif %}   
-        GROUP BY 1,2
-        
-        UNION ALL
-        SELECT _l1Token, _l2Token, NULL AS symbol, NULL AS decimals
-            FROM {{source( 'optimism_ethereum', 'OVM_L1StandardBridge_evt_ERC20DepositInitiated' ) }}
-        {% if is_incremental() %}
-        WHERE evt_block_time >= date_trunc("day", now() - interval '1 week')
-        {% endif %}   
-        GROUP BY 1,2
-
-        UNION ALL
-        SELECT l1_token, l2_token, symbol, decimals FROM {{ ref('ovm_optimism_l2_token_factory') }} 
-        {% if is_incremental() %}
-        WHERE call_block_time >= date_trunc("day", now() - interval '1 week')
-        {% endif %}   
-        GROUP BY 1,2,3,4
-
-        -- Manual adds
-        UNION ALL SELECT '0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f' AS l1_token, '0x8700daec35af8ff88c16bdf0418774cb3d7599b4' AS l2_token, NULL, NULL -- SNX
-        UNION ALL SELECT '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' AS l1_token,'0xdeaddeaddeaddeaddeaddeaddeaddeaddead0000' AS l2_token, NULL, NULL -- ETH
-        UNION ALL SELECT '0x6b175474e89094c44da98b954eedeac495271d0f' AS l1_token,'0xda10009cbd5d07dd0cecc66161fc93d7c9000da1' AS l2_token, NULL, NULL -- DAI
-        UNION ALL SELECT '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' AS l1_token, '0x4200000000000000000000000000000000000006' AS l2_token, NULL, NULL --WETH
-
-    ) map
+SELECT _l1Token AS l1_token, _l2Token AS l2_token, NULL AS symbol, NULL AS decimals
```

Comment:
do we need to explicitly cast null on these columns?

Reply:
how will trino default them?

Reply:
I had issues with datatypes before, so I think explicitly casting makes sense